homeassistant/components/mqtt/alarm_control_panel.py

```diff
@@ -266,79 +275,86 @@ async def async_alarm_disarm(self, code: str | None = None) -> None:
 
         This method is a coroutine.
         """
-        code_required = self._config[CONF_CODE_DISARM_REQUIRED]
-        if code_required and not self._validate_code(code, "disarming"):
+        if self._config[CONF_CODE_DISARM_REQUIRED] and not self._validate_code(
+            code, "disarming"
+        ):
```

Comment:
I think the previous style was better, fits on two lines instead of three (apply to all below):
```suggestion
        code_required: bool = self._config[CONF_CODE_DISARM_REQUIRED]
        if code_required and not self._validate_code(code, "disarming"):
```